ydb/Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [package]
-publish=true
+publish = true
```

Comment:
I use VSCode, rust code formatted with rust-analyzer, toml with Even Better Toml (I think so). I'll try to investigate how to do format in cli

Reply:
taplo-cli can format toml-files very good. Better toml uses it.
It can be installed or via package manager as binary package or via cargo install taplo-cli

to format Cargo.toml we can use, for example, 

```
 taplo fmt --option indent_string="    "  Cargo.toml 
```

options can be located in separate config file or placed into the command string.